src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -23,4 +23,35 @@ public int divide(int a, int b) throws IllegalArgumentException {
         }
         return a / b;
     }
+
```

Comment:
4. The method `subtract(int a, int b, int c)` has an unused parameter `c`. Either remove the unused parameter or use it within the method implementation.